src/dmd/tokens.d

```diff
@@ -451,6 +451,74 @@ extern (C++) struct Token
         Identifier ident;
     }
 
+    bool opEquals()(auto ref const Token rhs) const
```

Comment:
Don't use `auto ref`. Don't need to make this a template. Use `ref`.

Reply:
Ok.

Reply:
@WalterBright if it's `ref` and not `auto ref` and a template it doesn't work with temporaries.